link.go

```diff
@@ -53,9 +53,11 @@ type VfInfo struct {
 	Mac       net.HardwareAddr
 	Vlan      int
 	Qos       int
-	TxRate    int
+	TxRate    int // IFLA_VF_TX_RATE  Max TxRate
```

Comment:
This looks like some remaining after refactor, as max tx rate is defined also below and later in code only `MaxTxRate` is used, without no further mention of plain `TxRate`.
Am I correct there?

Reply:
The comment is here to clarify that TxRate maps to VF_TX_RATE netlink attribute.
This commit added MinTxRate and MaxTxRate which map to VF_RATE netlink attribute.

Reply:
ACK. Now it's clear for me.